lib/quail/quail/common/accessibility_tests.php

```diff
@@ -2908,17 +2908,59 @@ function check()
 
 		if (strlen($document_string) > $doc_length){
 
-			$no_headings = 0;
+			// Assume we have headings, look for all possible heading types
```

Comment:
This comment should probably read "Assume we **don't** have headings, look for all possible heading types".
Think that was just a typo on my part